### PR TITLE
Add CollectorSystems use case and requirements.

### LINKED ISSUE #5475: Document collector systems use case
See https://github.com/pulibrary/figgy/pull/5289

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #5475

### CODE REVIEW

File: docs/ux/use_cases.md [hunk 1]
```diff
@@ -62,3 +62,19 @@ When viewers support
...
+## CollectorSystems
+
+PUL Preservation & Conservation uses
+[CollectorSystems](https://www.collectorsystems.com/index.htm) to manage their
+workflows. They've worked with a developer there to add custom import processes
+into CollectorSystems. APIs it requires are:
+
+1. Fetch all Figgy objects linked to a source metadata identifier which are
+   restricted
+   * https://figgy.princeton.edu/catalog?q=source_metadata_identifier_ssim:9934953393506421&format=json&f[visibility_ssim][]=restricted
```
Comment (hackartisan) [inline docs/ux/use_cases.md:75]: Do they use a token for this?

Comment (tpendragon) [reply docs/ux/use_cases.md:75]: Yes! I removed it from the docs, but mentioning that somehow would be good.

Comment (tpendragon) [reply docs/ux/use_cases.md:75]: Added auth token info.

**Comment by escowles** [comment]
@tpendragon This looks correct and complete to me, for their usage of Figgy. They also pull metadata from Bibdata and Pulfalight — do you want to include that here?

**Comment by tpendragon** [comment]
@escowles Does that look better?

**Comment by escowles** [comment]
👍🏻 Look good to me

**Review by hackartisan — APPROVED** [review]

**Pull request merged**